source/_integrations/tesla_fleet.markdown

```diff
@@ -24,7 +24,7 @@ The Tesla Fleet API {% term integration %} exposes various sensors from Tesla ve
 
 ## Prerequisites
 
-You must have a [Tesla](https://tesla.com) account and a Tesla vehicle, PowerWall, Solar, or Wall Connector.
+You must have a [Tesla](https://tesla.com) account and a Tesla vehicle, PowerWall, Solar, or Wall Connector, and must not have disabled the [My Home Assistant](/integrations/my/) integration.
```

Comment:
How is that a prerequisite?

Reply:
I have to set the redirect URL in the application details registered with Tesla, so there is an external constraint making the "my" integration required for OAuth redirects to work.

Reply:
In that case, we need to provide details on how people can set those up themselves.

Reply:
That's been merged into Dev and the docs for it are in next. This PR is just to reflect the current state and requirement for using the built in client ID.

Reply:
https://github.com/home-assistant/core/pull/123324